protocol/messenger_backup_handler.go

```diff
@@ -393,25 +393,7 @@ func (m *Messenger) requestCommunityKeysAndSharedAddresses(state *ReceivedMessag
 	}
 
 	if isEncrypted {
-		request := &protobuf.CommunityEncryptionKeysRequest{
-			CommunityId: syncCommunity.Id,
-		}
-
-		payload, err := proto.Marshal(request)
-		if err != nil {
-			return err
-		}
-
-		rawMessage := &common.RawMessage{
-			Payload:             payload,
-			Sender:              m.identity,
-			CommunityID:         community.ID(),
-			SkipEncryptionLayer: true,
-			MessageType:         protobuf.ApplicationMetadataMessage_COMMUNITY_ENCRYPTION_KEYS_REQUEST,
-		}
-
-		_, err = m.SendMessageToControlNode(community, rawMessage)
-
+		err = m.communityRequestEncryptionKeys(community, nil)
```

Comment:
Suggestion to make it clearer what this does
```suggestion
		err = m.requestCommunityEncryptionKeys(community, nil)
```